snapshots/education/2018-04-18/literacy_rates.feather.dvc

```diff
@@ -1,82 +1,27 @@
-      WDI as our base, and then extended coverage by adding literacy estimates from
-      the CIA Factbook, as well as several other
-      long-run series, as follows:
-
-
-
-      - Data before 1800: Buringh, E., & Van Zanden, J. L. (2009). Charting the “Rise
-      of the West”: Manuscripts and Printed
-      Books in Europe, A long-term perspective from the sixth through eighteenth centuries.
-      The Journal of Economic History.
-      Online [here](http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.553.9220&rep=rep1&type=pdf).
-      Observations before
-      1800 are plotted at the midpoint of the given time range (1475 refers to 1451–1500,
-      1550 refers to 1501-1600 etc.)
-
-
-      - Data for 1820 and 1870 (except for the US): Broadberry  and O'Rourke (2010)
-      – The Cambridge Economic History of Modern
-      Europe: Volume 1, 1700-1870
-
-
-      - Data for the US: National Center for Education Statistics, online [here](nces.ed.gov/naal/lit_history.asp).
-
-
-      - Global estimates for 1820-2000: van Zanden, J.L., et al. (eds.) (2014), How
-      Was Life?: Global Well-being since 1820,
-      OECD Publishing. Online [here](http://www.oecd.org/statistics/how-was-life-9789264214262-en.htm).
-
-
-      - Historical estimates for Latin America: OxLAD – Oxford Latin American Economic
-      History Data Base, online [here](http://moxlad-staging.herokuapp.com/home/en).
-
-
-      - Most recent estimates for high-income countries, as well as any available
-      estimates for 2016: [CIA World Factbook](https://www.cia.gov/library/publications/the-world-factbook/).
-
-
-
-
-      Further notes:
-
-
-      - All sources rely on the same conceptual definition, but in many cases sources
-      do not agree with one another. Because
-      of this, year to year changes should be interpreted with caution. You can read
-      more about literacy measurement here:
-      https://ourworldindata.org/how-is-literacy-measured
-
-
-      - The World Bank's WDI estimates correspond to UNESCO Institute for Statistics.
-      OxLAD estimates come from several underlying
-      sources (see original documentation for more details).
-
-
-      - For Paraguay in 1982, the OxLAD data source was favoured over the World Bank
-      (WDI) as the latter estimates literacy
-      rates at 78.46%, a much lower estimate compared to neighbouring years for which
-      there was data available.
-
-
-      - Sources for each country-year observation can be found in [this table.](http://ourworldindata.org/wp-content/uploads/2018/06/cross-country-literacy-sources-final.csv)
-    url: '- World Bank: https://data.worldbank.org/indicator/SE.ADT.LITR.ZS </br>
-      - CIA Factbook: https://www.cia.gov/library/publications/the-world-factbook/
-      </br> - Links to other sources below'
+  origin:
+    # Data product / Snapshot
+    title: Cross-country literacy rates - World Bank, CIA World Factbook, and other Sources
+    description: >
```

Comment:
The sources should be specified in `citation_full` (although there can be some overlap with the description, if it helps describe the dataset).